### PR TITLE
Added 1st version of Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by b-dantas**
Hi,

I'm using these canvas model into my project and to keep easy adopting an ubiquitous language, I've translated/localized those resources to Brazilian Portuguese. An example of localization is shifting from GDPR to LGPD (our local privacy regulation). Moreover, I didn't translate resources from previous version of V5 because I believe doesn't make sense.

I kept same file/directory structure, creating a `pt` folder below `translations` directory.

I'm open to make any change if necessary to have this PR approved.

Regards,
B. Dantas

### CODE REVIEW

**Review by NTCoding — APPROVED** [review]

**Comment by NTCoding** [comment]
Amazing. Thank you for your contribution.

**Comment by joaoasrosa** [comment]
Although I'm `pt-pt` it looks good to me! 👍

File: translations/pt/resources/model-traits-worksheet-PT.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Ficha de Características do Modelo
+
+É provável que um modelo seja descrito por várias características diferentes. Revise a lista abaixo e escolha qual se aplica ao Contexto em que você está trabalhando ou pense em suas próprias características.
```
Comment (emgsilva) [inline translations/pt/resources/model-traits-worksheet-PT.md:3]: Maybe you can use "contexto" (no capital letter).

**Comment by emgsilva** [comment]
pt-pt like @joaoasrosa, but I think this looks great, good job @b-dantas

I only made a very small suggestion, take a look.

**Review by yellowbrickc — CHANGES_REQUESTED** [review]
I am neither pt-pt nor pt-br, but I am the one fixing links 😀: can you please add the reference to the translations block on the main Readme? Thanks!

File: translations/pt/resources/model-traits-worksheet-PT.md (hunk 1 above)
Comment (b-dantas) [reply translations/pt/resources/model-traits-worksheet-PT.md:3]: No problem. The reason why I chose to capitalize was to highlight that word is related a DDD concept, but let's keep it in lowercase, being match with English original version.

**Comment by b-dantas** [comment]
> I am neither pt-pt nor pt-br, but I am the one fixing links 😀: can you please add the reference to the translations block on the main Readme? Thanks!

For sure @yellowbrickc. I added the reference into translation block on Readme and I took advantage to add a Summary section, bring easy to read this long doc.

**Review by yellowbrickc — APPROVED** [review]

**Pull request merged**